tcms/testcases/models.py

```diff
@@ -339,14 +339,14 @@ def add_cc(self, email_addrs):
     def get_cc_list(self):
         """ Return the whole CC list """
         if not self.cc_list:
-            return []
+            return [] #pylint:disable=Protected-access
```

Comment:
Dos this have anything to do with silencing the `form-field-help-text-used` ? It is not even in the right location.